Gemfile.lock

```diff
@@ -0,0 +1,245 @@
+GEM
```

Comment:
Please remove the lock file from the PR

Reply:
This command `bundle exec jekyll serve` is creating Gemfile.lock (removed it, but again its get created).

Reply:
Add it to .gitignore